app/Models/News.php

```diff
@@ -111,5 +111,10 @@ public function next()
     {
         return News::where('id', '>', $this->id)->orderBy('id','asc')->first();
     }
+    
+    public function shortDecription()
+    {
+        return substr_replace(News::find($this->id)->description, '...', 100, -1);
```

Comment:
you don't need News::find($this->id)
you can access to description via $this->description